pkg/operator/starter.go

```diff
@@ -282,12 +289,14 @@ func RunOperator(ctx context.Context, controllerConfig *controllercmd.Controller
 	dynamicInformers.Start(ctx.Done())
 	migrationInformer.Start(ctx.Done())
 	migrationInformer.Start(ctx.Done())
```

Comment:
unrelated (but pleaes fix in a separate PR: why migrationInformer twice?

Reply:
Yes. raised a PR https://github.com/openshift/cluster-openshift-apiserver-operator/pull/355/files